yafti/plugin/run.py

```diff
@@ -89,6 +89,18 @@ async def exec(self, cmd: str) -> subprocess.CompletedProcess:
             cmd, returncode=proc.returncode, stdout=stdout, stderr=stderr
         )
 
+    async def install(self, package: str) -> YaftiPluginReturn:
+        """Execute a command on the host system
+
+        Args:
+          package: The command to execute
+
+        Returns:
+          An object containing the stdout and stderr from the command
+        """
+        cmd = package
+        return await self.exec(cmd)
```

Comment:
This can be simplified as 
```suggestion
        return await self.exec(package)
```